src/components/learner-credit-management/styles/index.scss

```diff
@@ -37,3 +42,11 @@
     border: 1px solid $danger;
   }
 }
+
+.budget-overview-card {
+  .budget-utilization-container {
+    .collapsible-body {
+      padding: 0;
```

Comment:
Any way to avoid custom class names solely for a `padding: 0`? E.g., is it possible to pass a `className="p-0"` to the appropriate component?

Reply:
`div` rendered by the Collapsible component isn't accessible, maybe might be accessible if I use advance option, but felt its might be overkill since I'm only trying to align the available budget section and utilisation section.

Reply:
Ah, you're right that the `.collapsible-body` element isn't accessible using the default `Collapsible` and agree its not worth refactoring to adopt the `Collapsible.Advanced` pattern here.

That said, it looks like you could consider using some negative margins to achieve the same effect on the parent `Stack` rendered as the child of `Collapsible`.

```jsx
return (
    <Collapsible
      className='mt-4'
      styling="basic"
      title={<h6 className="mb-0">Utilization details</h6>}
    >
      <Stack className="mt-2 mx-n2">
      // ...
    </Collapsible>
```

where `mx-n2` indicates negative left/right margin using the same spacer value as the `.collapsible-body` class name, effectively negating the `.collapsible-body` padding without any custom class names.